### ISSUE TITLE
holoviews.core.io.Pickler raises same 'Could not pickle custom style information.' warn for each Curve in object

### ISSUE THREAD

**Issue by canavandl** [ioam/holoviews#342]
```python
>> print(repr(holoviews_object)
:Layout
   .XIC.RT :HoloMap   [Compound]
      :NdOverlay   [Group ID,Peak ID,m/z]
         :Curve   [RT]   (Intensity)
   .XIC.MZ :HoloMap   [Compound]
      :NdOverlay   [m/z]
         :Curve   [m/z]   (Intensity)
```

Attempting to pickle the above object using holoviews.core.io.Pickler raises a warning for each element inside the object, yielding (potentially) a massive number of warns.

```python
>> from holoviews.core.io import Pickler
>> Pickler.save(holoviews_object, 'out.pkl')
WARNING:root:NdOverlay46623: Could not pickle custom style information.
WARNING:root:Curve41932: Could not pickle custom style information.
WARNING:root:Curve41938: Could not pickle custom style information.
WARNING:root:Curve41943: Could not pickle custom style information.
WARNING:root:Curve41977: Could not pickle custom style information.
... (continues for 600 warns)
```

**Comment by jbednar** [comment]
Looks like two issues -- making the style issue pickable, and making sure that users only get one warning in such cases, per Element type.

**Comment by canavandl** [comment]
This issue appears to be newly resolved in master.

**Comment by jlstevens** [comment]
I believe this is fixed in 54dfec8 (and similarly in 7c4c595 for ``HeatMap``).
